src/hook/useCollection.js

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
```

Comment:
@ibimina Do you think it will be helpful to actually consume this fetched data somewhere in the app? So we can see it? These are the mock listings right?

Reply:
ok